sigs.yaml

```diff
@@ -0,0 +1,31 @@
+sigs: []
```

Comment:
I could not really find some docs in the k8s community about what is done with the `sigs.yaml` file. Do you know?

Reply:
it is used to create a bunch of .md docs

Reply:
we need to define the sigs chair and tech lead then, like https://github.com/thoth-station/core/blob/master/community/sigs.yaml

Reply:
> we need to define the sigs chair and tech lead then

I believe that's the purporse of #2 and https://github.com/goern/open-services-group-community/pull/1